### ISSUE TITLE
Feature Request - Policy Groups

### ISSUE THREAD

**Issue by ghost** [tchx84/Flatseal#567]
**Policy Groups would empower users to efficiently configure specific applications in bulk, allowing them to customize which applications are included within these groups**. For instance, I often find the need to uniformly manage all QT applications.

The key concept here is prioritization. We assume that user-specific configurations for individual applications should hold the highest priority. Policy Groups would come next in priority, with global application configurations having the lowest priority.

In other words, modifications made at the individual application level should not be overridden by Policy Group configurations. Likewise, Policy Group configurations should remain unaffected by global application configuration changes.
 
Thank you for considering this suggestion. Your dedication to improving the project is greatly appreciated. 😊

**Comment by rusty-snake** [comment]
Will your idea sounds very useful, keep in mind that flatseal is just a frontend for flatpak's sandbox permissions. Will it would be technical possible to provide a superset of the features of `flatpak override` like automation, "templates" or "template-groups". It is not possible to change the way flatpak reads the sandbox configurations or how it builds the sandbox.

**Comment by ghost** [comment]
> Will your idea sounds very useful, keep in mind that flatseal is just a frontend for flatpak's sandbox permissions. Will it would be technical possible to provide a superset of the features of `flatpak override` like automation, "templates" or "template-groups". It is not possible to change the way flatpak reads the sandbox configurations or how it builds the sandbox.

Maybe my expression is not accurate. The configuration here i mean is just what flatseal is doing now, basically just `flatpak override`. But I wish i can toggle them by custom groups.
